src/coreclr/vm/stringliteralmap.cpp

```diff
@@ -357,7 +357,7 @@ void GlobalStringLiteralMap::Init()
         ThrowOutOfMemory();
 }
 
-StringLiteralEntry *GlobalStringLiteralMap::GetStringLiteral(EEStringData *pStringData, DWORD dwHash, BOOL bAddIfNotFound)
+StringLiteralEntry *GlobalStringLiteralMap::GetStringLiteral(EEStringData *pStringData, DWORD dwHash, BOOL bAddIfNotFound, BOOL bAppDomainWontUnload)
```

Comment:
Nit: `bAppDomainWontUnload` -> `bIsCollectible` to make the naming consistent with what we use in other places.